WooCommerce/Classes/Authentication/Jetpack Setup/Site Credential Login/SiteCredentialLoginViewModel.swift

```diff
@@ -71,9 +78,11 @@ extension SiteCredentialLoginViewModel: LoginFacadeDelegate {
         let wrongCredentials = err.domain == Constants.xmlrpcErrorDomain && err.code == Constants.invalidCredentialErrorCode
         errorMessage = wrongCredentials ? Localization.wrongCredentials : Localization.genericFailure
         shouldShowErrorAlert = true
+        analytics.track(.loginJetpackSiteCredentialDidShowErrorAlert)
```

Comment:
Similar as above: 
```suggestion
        analytics.track(.loginJetpackSiteCredentialDidShowErrorAlert, withError: error)
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/8187/commits/07753df41dfa10ee9c64de7d2d3e2730596270c2